runtime/sema/authaccount_type.go

```diff
@@ -690,6 +703,36 @@ var AccountKeysTypeGetFunctionType = &FunctionType{
 	RequiredArgumentCount: RequiredArgumentCount(1),
 }
 
+// fun forEach(_ function: (AccountKey: Bool)): Void
+var AccountKeysTypeForEachFunctionType = func() *FunctionType {
+	// fun (_ key: AccountKey): Bool
+	iterFunctionType := &FunctionType{
+		Parameters: []*Parameter{
+			{
+				Identifier:     "accountKey",
+				TypeAnnotation: NewTypeAnnotation(AccountKeyType),
+			},
+		},
+		ReturnTypeAnnotation: NewTypeAnnotation(BoolType),
+	}
+
+	return &FunctionType{
+		Parameters: []*Parameter{
+			{
+				Label:          ArgumentLabelNotRequired,
+				Identifier:     "function",
+				TypeAnnotation: NewTypeAnnotation(iterFunctionType),
+			},
+		},
+		ReturnTypeAnnotation: NewTypeAnnotation(VoidType),
+	}
+}()
+
+var AccountKeysTypeCountFunctionType = &FunctionType{
+	Parameters:           []*Parameter{},
+	ReturnTypeAnnotation: NewTypeAnnotation(IntType),
+}
```

Comment:
This is a field, not a function

```suggestion
var AccountKeysTypeCountFieldType = IntType
```